tools/src/main/resources/operatorsScore.csv

```diff
@@ -20,6 +20,7 @@ ObjectHashAggregateExec,3.0
 SortAggregateExec,3.0
 InMemoryTableScanExec,3.0
 DataWritingCommandExec,3.0
+ExecutedCommandExec,3.0
```

Comment:
was this based on performance results?  ie delta writes acceleration seems about same as like parquet writes?

Reply:
This was not, it's just the default setting that was automatically added by the doc generation.  I added a note in #7294 that the score may need to be tuned.